Anima/math-tools/multi_compartment_base/animaMCMConstants.h

```diff
@@ -28,4 +29,58 @@ const double DiffusionSmallDelta = 10.0e-3;
 //! Default big delta value (classical values)
 const double DiffusionBigDelta = 40.0e-3;
 
+//! Default zero lower bound (in case we want something else than zero one day)
+const double MCMZeroLowerBound = 0.0;
+
+//! Epsilon value in case we do not want to have parameters reaching their true bounds (used for now in DDI)
```

Comment:
Not anymore.

Reply:
Yes but that's not on the master yet, so until then I keep it